src/System.Net.Http/tests/FunctionalTests/HttpResponseMessageTest.cs

```diff
@@ -120,13 +120,13 @@ public void EnsureSuccessStatusCode_VariousStatusCodes_ThrowIfNot2xx()
         }
 
         [Fact]
-        public void EnsureSuccessStatusCode_AddContentToMessage_ContentIsDisposed()
+        public void EnsureSuccessStatusCode_AddContentToMessage_ContentIsNotDisposed()
```

Comment:
Does this need to be skipped now on netfx, since it doesn't have the change?

Reply:
Yes.  Forgot about that. I'll update the test.